model_tools/brain_transformation/__init__.py

```diff
@@ -41,8 +41,10 @@ def __init__(self, identifier,
         behavioral_readout_layer = behavioral_readout_layer or layers[-1]
         probabilities_behavior = ProbabilitiesMapping(identifier=identifier, activations_model=activations_model,
                                                       layer=behavioral_readout_layer)
+        odd_one_out_behavior = OddOneOutBehavior(identifier=identifier, activations_model=activations_model)
```

Comment:
```suggestion
        odd_one_out_behavior = OddOneOutBehavior(identifier=identifier, activations_model=activations_model,
                                                                                    layer=behavioral_readout_layer)
```
You'll also need to know which layer to use